sonar-db/src/main/java/org/sonar/db/dashboard/WidgetPropertyDao.java

```diff
@@ -73,14 +73,13 @@ public WidgetPropertyDto selectByKey(DbSession session, Long propertyId) {
     return mapper(session).selectByDashboard(dashboardKey);
   }
 
-  public void deleteByWidgetIds(final DbSession session, List<Long> widgetIdsWithPropertiesToDelete) {
-    DatabaseUtils.executeLargeInputs(widgetIdsWithPropertiesToDelete, new Function<List<Long>, List<Void>>() {
-      @Override
-      public List<Void> apply(List<Long> input) {
-        mapper(session).deleteByWidgetIds(input);
-        return Arrays.asList();
-      }
-    });
+  public void deleteByWidgetIds(DbSession session, List<Long> widgetIdsWithPropertiesToDelete) {
+    executeLargeInputs(
+        widgetIdsWithPropertiesToDelete,
+        input -> {
+          mapper(session).deleteByWidgetIds(input);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make this line start at column 7. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AIndentationCheck)